kt/godot-runtime/src/main/kotlin/godot/core/KtClass.kt

```diff
@@ -8,7 +8,8 @@ class KtClass<T : KtObject>(
         private val constructors: Map<Int, KtConstructor<T>>,
         private val _properties: Map<String, KtProperty<T, *>>,
         private val _functions: Map<String, KtFunction<T, *>>,
-        private val _signalInfos: Map<String, KtSignalInfo>
+        private val _signalInfos: Map<String, KtSignalInfo>,
+        val baseGodotClass: String
```

Comment:
I think we should add `@Suppress("UNUSED_VARIABLE") ` to each variable accessed via jni from cpp.

Reply:
I know it's more work but i my idea was to just add it to the variables that are used from cpp. Not just everything. The reason being: to be able to see in the ide if it really is used or not and kinda as documentation as what variables are accessed through jni.
If you don't agree I'm also fine with just leaving it like this.